modules/configure-cluster-decommissioning.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * configuration/cluster-decommissioning.adoc
+:_content-type: PROCEDURE
+[id="configure-cluster-decommissioning_{context}"]
+= Configuring cluster decommissioning
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *System Configuration*.
+. In the *System Configuration* header, click *Edit*.
+. In the *Cluster deletion* section, you can configure the following fields:
+* *Decommissioned cluster age*: The number of days that a cluster is unreachable before it is considered for deletion. If Central cannot reach Sensor on the cluster for this number of days, the cluster and all of its resources are automatically deleted. To leave this feature disabled, which is the default behavior, enter 0 in this field. To enable this feature, enter a number, such as 90, to configure the number of unreachable days. 
```

Comment:
Suggest adding "a `nonzero` number" per your instruction in the earlier paragraph.